pkg/schemas/oauth_security_integration_for_partner_applications.go

```diff
@@ -30,33 +31,36 @@ var DescribeOauthIntegrationForPartnerApplications = map[string]*schema.Schema{
 	"oauth_allowed_token_endpoints":         DescribePropertyListSchema,
 }
 
+var OauthIntegrationForPartnerApplicationsPropertiesNames = []string{
+	"OAUTH_CLIENT_TYPE",
+	"OAUTH_REDIRECT_URI",
+	"ENABLED",
+	"OAUTH_ALLOW_NON_TLS_REDIRECT_URI",
+	"OAUTH_ENFORCE_PKCE",
+	"OAUTH_USE_SECONDARY_ROLES",
+	"PRE_AUTHORIZED_ROLES_LIST",
+	"BLOCKED_ROLES_LIST",
+	"OAUTH_ISSUE_REFRESH_TOKENS",
+	"OAUTH_REFRESH_TOKEN_VALIDITY",
+	"NETWORK_POLICY",
+	"OAUTH_CLIENT_RSA_PUBLIC_KEY_FP",
+	"OAUTH_CLIENT_RSA_PUBLIC_KEY_2_FP",
+	"COMMENT",
+	"OAUTH_CLIENT_ID",
+	"OAUTH_AUTHORIZATION_ENDPOINT",
+	"OAUTH_TOKEN_ENDPOINT",
+	"OAUTH_ALLOWED_AUTHORIZATION_ENDPOINTS",
+	"OAUTH_ALLOWED_TOKEN_ENDPOINTS",
+}
+
 func DescribeOauthIntegrationForPartnerApplicationsToSchema(integrationProperties []sdk.SecurityIntegrationProperty) map[string]any {
 	securityIntegrationProperties := make(map[string]any)
 	for _, property := range integrationProperties {
 		property := property
-		switch property.Name {
-		case "OAUTH_CLIENT_TYPE",
-			"OAUTH_REDIRECT_URI",
-			"ENABLED",
-			"OAUTH_ALLOW_NON_TLS_REDIRECT_URI",
-			"OAUTH_ENFORCE_PKCE",
-			"OAUTH_USE_SECONDARY_ROLES",
-			"PRE_AUTHORIZED_ROLES_LIST",
-			"BLOCKED_ROLES_LIST",
-			"OAUTH_ISSUE_REFRESH_TOKENS",
-			"OAUTH_REFRESH_TOKEN_VALIDITY",
-			"NETWORK_POLICY",
-			"OAUTH_CLIENT_RSA_PUBLIC_KEY_FP",
-			"OAUTH_CLIENT_RSA_PUBLIC_KEY_2_FP",
-			"COMMENT",
-			"OAUTH_CLIENT_ID",
-			"OAUTH_AUTHORIZATION_ENDPOINT",
-			"OAUTH_TOKEN_ENDPOINT",
-			"OAUTH_ALLOWED_AUTHORIZATION_ENDPOINTS",
-			"OAUTH_ALLOWED_TOKEN_ENDPOINTS":
+		if slices.Contains(OauthIntegrationForPartnerApplicationsPropertiesNames, property.Name) {
 			securityIntegrationProperties[strings.ToLower(property.Name)] = []map[string]any{SecurityIntegrationPropertyToSchema(&property)}
-		default:
-			log.Printf("[WARN] unexpected property %v returned from Snowflake", property.Name)
+		} else {
+			log.Printf("[WARN] unexpected property %v in oauth security integration returned from Snowflake", property.Name)
```

Comment:
maybe just for clarity add a type (partner application / custom clients) here and in the other oauth integration, so we'll know exactly where we "failed".

Reply:
Done.